src/Components/Web/src/Forms/InputBase.cs

```diff
@@ -50,6 +50,11 @@ public abstract class InputBase<TValue> : ComponentBase, IDisposable
         /// </summary>
         [Parameter] public Expression<Func<TValue>>? ValueExpression { get; set; }
 
+        /// <summary>
+        /// Gets or sets the display name for this field.
```

Comment:
Could you include a snippet that says what it's used for (e.g. *This value is used when generating error messages when the input value fails to parse correctly.*)